pkg/deploy/devconfig.go

```diff
@@ -136,7 +136,7 @@ func DevConfig(_env env.Core) (*Config, error) {
 				adminKeyvaultAccessPolicy(_env),
 				deployKeyvaultAccessPolicy(_env),
 			},
-			FluentbitImage:       to.StringPtr(os.Getenv("FLUENTBIT_IMAGE")),
+			FluentbitImage:       to.StringPtr(version.FluentbitImage(os.Getenv("USER") + _env.Environment().ContainerRegistryDNSSuffix)),
```

Comment:
nice magic. 
public - ContainerRegistryDNSSuffix:   "azurecr.io",
ff - ContainerRegistryDNSSuffix:   "azurecr.us",
any other cloud would have this undefined?

Reply:
Yeah that is correct.  This should be pulled from the azure environment struct provided by the azure-sdk-for-go so that's the intention.